subsys/bluetooth/controller/ll_sw/ull_llcp_local.c

```diff
@@ -151,6 +151,9 @@ void llcp_lr_rx(struct ll_conn *conn, struct proc_ctx *ctx, struct node_rx_pdu *
 	case PROC_DATA_LENGTH_UPDATE:
 		llcp_lp_comm_rx(conn, ctx, rx);
 		break;
+	case PROC_CTE_REQ:
```

Comment:
the case is within CONFIG_BT_CTRL_DATA_LENGTH define?
Should be under CTE_REQ define?

Reply:
No, that is wrong. Corrected and added #define guards.